R/utils.R

```diff
@@ -99,7 +99,7 @@ parse_ws_url <- function(ws_url) {
   # NOTE: ws_url must be a character scalar
   ws_url <- httr::parse_url(ws_url)
   # ws_url scheme must be ws or wss:
-  if(!identical(ws_url$scheme, "ws") && !identical(ws_url, "wss")) {
+  if(!identical(ws_url$scheme, "ws") && !identical(ws_url$scheme, "wss")) {
```

Comment:
Good catch!

Reply:
it helps finding it by creating tests ! ☺️